code/modules/events/disease_outbreak.dm

```diff
@@ -98,66 +172,153 @@
+		if(chosen_symptom)
+			var/datum/symptom/new_symptom = new chosen_symptom
+			symptoms += new_symptom
+	Refresh()
 
-/datum/event_admin_setup/disease_outbreak/advanced
-	///Admin selected custom severity rating for the event
-	var/max_severity
-	///Admin selected custom value for the maximum symptoms this virus should have
-	var/max_symptoms
+	name = "Sample #[rand(1, 9999)]"
 
-/datum/event_admin_setup/disease_outbreak/advanced/prompt_admins()
-	var/candidate_count = candidate_check()
-	if(!candidate_check())
-		tgui_alert(usr, "There are no candidates eligible to recieve a disease!", "Error")
-		return ADMIN_CANCEL_EVENT
-	tgui_alert(usr, "[candidate_count] candidates found!", "Disease Outbreak")
+// Assign the properties for the virus
+/datum/disease/advance/random/event/AssignProperties()
+	visibility_flags |= HIDDEN_SCANNER
 
-	if(tgui_alert(usr,"Customize your virus?", "Glorified Debug Tool", list("Yes", "No")) == "Yes")
-		max_severity = tgui_input_number(usr, "Select a custom severity for your virus!", "Plague Incorporation!", 3, 8)
-		max_symptoms = tgui_input_number(usr, "How many symptoms do you want your virus to have?", "A pox upon ye!", 3, 15)
+	//Spread method this virus should have
+	var/transmissibility = rand(1, 100)
 
-/datum/event_admin_setup/disease_outbreak/advanced/apply_to_event(datum/round_event/disease_outbreak/advanced/event)
-	event.max_severity = max_severity
-	event.max_symptoms = max_symptoms
+	addtimer(CALLBACK(src, PROC_REF(MakeVisible)), ((ADV_ANNOUNCE_DELAY * 2) - 10) SECONDS) // Keep the virus hidden until the timer expires
+
+	if(properties?.len)
+		spreading_modifier = max(CEILING(0.4 * properties["transmittable"], 1), 1)
+		cure_chance = clamp(7.5 - (0.5 * properties["resistance"]), 5, 10) // Can be between 5 and 10
+		stage_prob = max(0.5 * properties["stage_rate"], 1)
+		SetSeverity(properties["severity"])
+		GenerateCure(properties)
+		if(transmissibility < ADV_SPREAD_LOW)
+			SetSpread(DISEASE_SPREAD_CONTACT_FLUIDS)
+
+		else if(transmissibility < ADV_SPREAD_MID)
+			SetSpread(DISEASE_SPREAD_CONTACT_SKIN)
+
+		else
+			SetSpread(DISEASE_SPREAD_AIRBORNE)
+
+	else
+		CRASH("Our virus properties were empty or null!")
+
+// Reveal the virus when the level 7 announcement happens
+/datum/disease/advance/random/event/proc/MakeVisible()
+	visibility_flags &= ~HIDDEN_SCANNER
+
+// Assign the spread type and give it the correct description
+/datum/disease/advance/random/event/SetSpread(spread_id)
+	switch(spread_id)
+		if(DISEASE_SPREAD_CONTACT_FLUIDS)
+			spread_flags = DISEASE_SPREAD_BLOOD | DISEASE_SPREAD_CONTACT_FLUIDS
+			spread_text = "Fluids"
+		if(DISEASE_SPREAD_CONTACT_SKIN)
+			spread_flags = DISEASE_SPREAD_BLOOD | DISEASE_SPREAD_CONTACT_FLUIDS | DISEASE_SPREAD_CONTACT_SKIN
+			spread_text = "On contact"
+		if(DISEASE_SPREAD_AIRBORNE)
+			spread_flags = DISEASE_SPREAD_BLOOD | DISEASE_SPREAD_CONTACT_FLUIDS | DISEASE_SPREAD_CONTACT_SKIN | DISEASE_SPREAD_AIRBORNE
+			spread_text = "Airborne"
```

Comment:
```suggestion
			spread_text = "fluids"
		if(DISEASE_SPREAD_CONTACT_SKIN)
			spread_flags = DISEASE_SPREAD_BLOOD | DISEASE_SPREAD_CONTACT_FLUIDS | DISEASE_SPREAD_CONTACT_SKIN
			spread_text = "contact"
		if(DISEASE_SPREAD_AIRBORNE)
			spread_flags = DISEASE_SPREAD_BLOOD | DISEASE_SPREAD_CONTACT_FLUIDS | DISEASE_SPREAD_CONTACT_SKIN | DISEASE_SPREAD_AIRBORNE
			spread_text = "airborne"
```

i have another change to the message we spit out to the world that will make this look a lot more natural.